### ISSUE TITLE
"Bills to pay" summary ignores currencies

### ISSUE THREAD

**Issue by wgslr** [firefly-iii/firefly-iii#3475]
**Bug description**
I am running Firefly III version 5.2.8 and my problem is:

"Bills to pay" summary in the dashboard displays a sum of bills as if they were all in my default currency, even though they are not.

**Steps to reproduce**
1. Create bills in EUR and PLN currencies
2. Observe correctly split sum in the bills view:
![correct-sum-of-bills](https://user-images.githubusercontent.com/6932414/85200518-5c655600-b2f8-11ea-97b2-2c0c89c38b5f.png)

3. Go to the dashboard
4. Observe sum of the values displayed (zł is my default currency)

![bad-sum-of-bills](https://user-images.githubusercontent.com/6932414/85200527-71da8000-b2f8-11ea-89f4-cf52f8fedc02.png)

5. Change defualt currency to EUR
6. See the currency change, with no change in values

![bills-to-pay-euro](https://user-images.githubusercontent.com/6932414/85200592-0d6bf080-b2f9-11ea-89e8-e7f490f58c92.png)

**Expeced behaviour**

Display amount for each currency separately, as it is done in "Net worth" field and other.

**Bonus points**

- I searched and nobody reported this bug before **yes**
- I have added a screenshot. **yes**
- I was able to replicate it on the demo site https://demo.firefly-iii.org/ **yes**

**Comment by JC5** [comment]
Indeed! I recently noticed something similar. Thanks for letting me know, I'll write it down!

**Comment by wgslr** [comment]
Thank you @JC5  for a quick response. And I realized I have a related (currency & bills themed) report to make. Let me now if I should move it to a separate issue:

1. I created a bill conifgured with "currency: EUR, expected amount: 12,29"
3. Modified the bill rule to only check description and not the currency&amount
3. I created an expense on a PLN account paying for the bill: 54,94 zł (native), 12,29 EUR (foreign)
4. Now the "average bill amount" uses PLN value and displays it as euro: €54.94

![image](https://user-images.githubusercontent.com/6932414/85206311-736c6e00-b321-11ea-8082-b3534c72dbe1.png)
![image](https://user-images.githubusercontent.com/6932414/85206316-78c9b880-b321-11ea-99e0-c077155ffcde.png)
